docker-compose.yml

```diff
@@ -0,0 +1,32 @@
+version: '3.8'
+
+services:
+  postgres:
+    image: postgres
+    environment:
+      POSTGRES_DB: dodo_db
+      POSTGRES_USER: postgres
+      POSTGRES_PASSWORD: postgres
```

Comment:
Please use env_file directive here. This should not be leaking into github.

Reply:
Set that.